README.md

```diff
@@ -271,6 +271,17 @@ Before starting these steps, perform [steps 1-5 from above](#setting-up-the-deve
   bundle exec rake assets:precompile
   ```
 
+ 1. Precompile the assets for Production Environment (else you may get squares instead of icons and other linking problems):
+
+  ```bash
+  RAILS_ENV=production rake assets:precompile
```

Comment:
Good catch! `rake assets:precompile` should have the `RAILS_ENV=production` environment set.

However, the `rake assets:precompile` is now there twice. This is unnecessary. Instead of adding a new step, could you just add the `RAILS_ENV=production` to the previous step?